File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/db2/NullableType.scala
package dev.typr.foundationssc.docs.db2
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object NullableType:
  //start
  val notNull: Db2Type[Int] = Db2Types.integer
  val nullable: Db2Type[Option[Int]] = Db2Types.integer.opt
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/db2/NullableType.kt
package dev.typr.foundationskt.docs.db2

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class NullableType {
    //start
    val notNull: Db2Type<Int> = Db2Types.integer
    val nullable: Db2Type<Int?> = Db2Types.integer.opt()
    //stop
}
